examples/integrations/oci/base/identity-se/oci-setup.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
```

Comment:
Please use `-e` or `set -e`. Also it's safer to just use bash for the shebang.

Reply:
Added -e